### PR TITLE
[ES-188] [ES-213] Hosting of mosip-plugins on GitHub pages in storybook based on branches

### PR DESCRIPTION
**Pull request by zesu22**
Branches based version hosting of mosip-plugins in the github pages

completed [ES-188]

and completed point 3 in [ES-213] `Host react-based mosip-plugins in GitHub pages storybook`
remaining part of [ES-213] will be taken care in sprint 6

Screenshot (238)

[ES-188]: https://mosip.atlassian.net/browse/ES-188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ES-213]: https://mosip.atlassian.net/browse/ES-213?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ES-213]: https://mosip.atlassian.net/browse/ES-213?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: storybook-example/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mosip/mosip-plugins"
+    "url": "git+https://github.com/mosip/mosip-plugins/storybook-example"
```
Comment (anshulv1401) [inline storybook-example/package.json:8]: remove the /storybook-example from the link. Link is not valid

File: storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx [hunk 2]
```diff
@@ -57,17 +58,17 @@ const sbiEnv = {
...
-  render: (args) => renderComponent(args),
+  component: SecureBiomtericInterfaceIntegrator,
   argTypes: {
-    container: {
-      control: "object",
-      type: { required: true, name: "HTMLElement" },
-      description:
-        "Container inside of which the whole component will be created",
-    },
+    //   container: {
```
Comment (anshulv1401) [inline storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx:66]: if not required, can we remove this commented parameter

File: storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx [hunk 3]
```diff
@@ -135,7 +135,7 @@ export default {
     },
     onCapture: {
       control: "events",
-      type: { required: true },
+      // type: { required: true },
```
Comment (anshulv1401) [inline storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx:138]: can we remove commented parameter

Comment (zesu22) [reply storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx]: resolved in this [ca7c463](https://github.com/mosip/mosip-plugins/pull/56/commits/ca7c463b799a7e2584b2318c9bd15a2fa66880e7)

File: storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx (hunk 2 above)
Comment (zesu22) [reply storybook-example/stories/SecureBiometricInterfaceIntegrator.stories.tsx]: resolved in this [ca7c463](https://github.com/mosip/mosip-plugins/pull/56/commits/ca7c463b799a7e2584b2318c9bd15a2fa66880e7)

File: storybook-example/package.json (hunk 1 above)
Comment (zesu22) [reply storybook-example/package.json]: resolved in this [ca7c463](https://github.com/mosip/mosip-plugins/pull/56/commits/ca7c463b799a7e2584b2318c9bd15a2fa66880e7)

**Review by anshulv1401 — APPROVED** [review]

**Review by anshulv1401 — APPROVED** [review]

**Pull request merged**